backend/src/admin/admin.controller.ts

```diff
@@ -21,17 +23,11 @@ export class AdminController {
     return this.adminService.login(userData);
   }
 
-  /** TEST CODE STARTS */
-  @Get('t1')
-  async getAll() {
-    return this.adminService.getAll1();
-  }
-
+  @UsePipes(new ValidationPipe())
   @UseGuards(AuthGuard('jwt'))
-  @Get('t2')
-  async getAll2() {
-    return this.adminService.getAll2();
+  @Post('register/junior')
+  async registerJunior(@Admin() payload: any, @Body() userData: RegisterJuniorDto) {
```

Comment:
Should this be in the junior controller?

Reply:
I would argue that as this method will only be executable by admins, it is better placed within the admin route; however, we should discuss this.